sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -970,6 +986,21 @@ public PDone apply(PCollection<TableRow> input) {
         return VoidCoder.of();
       }
 
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        if (table != null) {
+          builder.add("tableSpec", toTableSpec(table));
+        }
+
+        if (schema != null) {
+          builder.add("schema", schema.toString());
```

Comment:
this will be a JSON blob. Should it be prettified? Should we have monitoring support for prettification? any size limits?

(probably good as is)